crates/continuations/src/lib.rs

```diff
@@ -172,11 +193,16 @@ pub enum State {
     /// The `VMContRef` has been created, but `resume` has never been
     /// called on it. During this stage, we may add arguments using `cont.bind`.
     Allocated,
```

Comment:
Nitpick: I'd call this "fresh".

Reply:
Does it make sense to differentiate between a fresh and suspended continuation? Semantically you can view a fresh continuation as a suspended continuation.

Reply:
That's true, but we need to differentiate between the two for an ugly technical reason:
For an `Allocated/`Fresh` continuation, all `cont.bind` arguments must be written to the buffer that will be passed to the array call trampoline. For a `Suspended` continuation, it's another buffer.

It would be nice to get rid of the distinction between these two buffers once we rework payload passing eventually, at which point we should make the change you suggest.